webpack/assets/javascripts/react_app/components/SettingsTable/components/SettingValueEdit.js

```diff
@@ -72,6 +72,11 @@ const SettingValueEdit = ({ setting, updateSetting }) => {
   };
 
   const handleSubmit = () => {
+    if (value !== '' && value.trim() === '') {
```

Comment:
If I try to edit per_page setting I get SettingValueEdit.js:75 Uncaught TypeError: value.trim is not a function